changelog.txt

```diff
@@ -1,3 +1,42 @@
+14.0-RC2 (2024-03-09)
+------------------------------------------------------------------------
+Update: Bump bundled OpenTTD TTF fonts to version v0.6 (#12276)
+Update: Developer credits (#12173, #12235)
+Change: Use (at least) standard toolbar button size for signal selection buttons (#12265)
+Change: [Script] Match FormatString behaviour more closely (#12205)
+Fix #12255: Inconsistent punctuation in English localisation (#12262)
```

Comment:
Practically these are just inconsistencies in the translation, like any of the other language updates/backports.
I wouldn't specifically mention what got changed for English, but not all others. Especially since the change is so insignificant.